flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/tiered/common/TieredStorageConfiguration.java

```diff
@@ -344,6 +401,21 @@ public Builder setRemoteStorageBasePath(String remoteStorageBasePath) {
             return this;
         }
 
+        public Builder setMemorySafeModeEnabled(boolean memorySafeModeEnabled) {
+            this.memorySafeModeEnabled = memorySafeModeEnabled;
+            return this;
+        }
+
+        public Builder setMinBuffersPerGate(int minBuffersPerGate) {
+            this.minBuffersPerGate = minBuffersPerGate;
+            return this;
+        }
+
```

Comment:
We'd better also add the setter method for poolSizeCheckInterval in case of other usage.